packages/server/lib/server.ts

```diff
@@ -157,90 +154,93 @@ if (MANAGED_AUTH_ENABLED) {
 }
 
 // Webapp routes (session auth).
-app.route('/api/v1/meta').get(webAuth, environmentController.meta.bind(environmentController));
```

Comment:
signup/login/etc... routes above are not part of the web router?

Reply:
oh yes good catch